internal/factsengine/gatherers/verifypassword.go

```diff
@@ -73,7 +73,7 @@ func (g *VerifyPasswordGatherer) Gather(factsRequests []entities.FactRequest) ([
 
 		crypter := sha512crypt.New()
 		isPasswordWeak := false
-		var gatheringError *entities.FactGatheringError = nil
+		var gatheringError *entities.FactGatheringError
```

Comment:
Not a big fun of adding this off topic changes in this PR...
I wouldn't include them